src/mainwindow.cpp

```diff
@@ -56,10 +57,28 @@ MainWindow::MainWindow(QWidget *parent) :
     ui->bottomUpTreeView->sortByColumn(CostModel::SelfCost);
     ui->bottomUpTreeView->setModel(proxy);
 
+    auto topHotspotsProxy = new QSortFilterProxyModel(this);
```

Comment:
this sorts, and we need another level of proxy: a TopProxy (cf. heaptrack) that removes the child rows to make a flat table out of a tree. Additionally, it should hide the unwanted columns. So we have this chain:

view -> qsfpm -> topproxy -> m_bottomUpCostModel